FirebaseAdmin/FirebaseAdmin.Tests/Messaging/FirebaseMessagingClientTest.cs

```diff
@@ -89,6 +88,65 @@ public async Task SendAsync()
             Assert.Equal(2, handler.Calls);
         }
 
+        [Fact]
+        public async Task SendAllAsync()
```

Comment:
Also add unit tests for null list, empty list and list longer than 100. And a mock response that contains some errors.